src/elements/content-sidebar/activity-feed/task-new/messages.js

```diff
@@ -51,14 +51,19 @@ const messages = defineMessages({
     },
     tasksFeedStatusLabel: {
         id: 'be.contentSidebar.activityFeed.taskNew.tasksFeedStatusLabel',
-        defaultMessage: 'Status: {taskStatus}',
+        defaultMessage: '{taskStatus}',
```

Comment:
Is this message still needed or can the status value be rendered directly?

Reply:
Note: we may need to audit this from an a11y perspective. Maybe the old string should be the aria-label for the pill.